wherehows-web/app/templates/components/dataset-aclaccess.hbs

```diff
@@ -82,7 +82,11 @@
           </button>
           <div class="ember-power-calendar-nav-title">
             {{moment-format calendar.center 'MMMM YYYY'}}
-            <p>Expires in {{moment-from-now calendar.selected}}</p>
+
+            <p>
+              {{if hasValidExpiration (concat "Expires in " (moment-from-now calendar.selected)) ""}}
```

Comment:
Why are you concat-ing an empty string at the end? Seems kind of pointless.

Reply:
👍